### DISCUSSION TITLE
Trans flag and symbol missing image replacements

### DISCUSSION THREAD

**Discussion by Lunarequest** [community/community#5418] [Other Feature Feedback, Questions, & Ideas]
Currently if you use firefox on linux emoji's are switched to .png replacements on page load. While this is not an issue in most cases for almost a month 🏳️‍⚧️ and ⚧️  have been missing their png counterparts which makes some pages such as my profile incredibly ugly.

This behavior is always triggered on firefox when running on linux even if the font used supports these emojis. switching your browser agent with an [extentions](https://addons.mozilla.org/en-US/firefox/addon/uaswitcher/) shows the page using the current system font and working as indented.  Please fix your fallback images or make the emoji -> png conversion  "opt-in"

**Comment by rcassani** [comment]
@Lunarequest, meanwhile this issue is updated, there is the [GitHub System Emoji](https://github.com/wezm/github-system-emoji-extension) Firefox Add-On.

**Reply by Lunarequest** [reply]
This is what I was looking for thank you.
